maraboupy/MarabouCore.cpp

```diff
@@ -136,6 +138,53 @@ bool createInputQuery(InputQuery &inputQuery, std::string networkFilePath, std::
   return true;
 }
 
+std::list<unsigned> getParticipatingVariables( const Equation &equation )
```

Comment:
So, I'm a little confused as to why this is needed. I saw you were just following what's currently going on, but do you know why this is even exposed?